tests/integration/components/cf-form-test.js

```diff
@@ -140,6 +146,11 @@ module("Integration | Component | cf-form", function(hooks) {
       slug: "file-question",
       type: "FILE"
     });
+    this.server.create("question", {
+      formIds: [form.id],
+      slug: "date-question",
+      type: "DATE"
+    });
```

Comment:
I suppose this is commented out because it's causing issues? Maybe add a comment explaining the place where you got stuck?

Reply:
Yes. That's the 'skip'. I'll add a comment.